bundles/org.openhab.binding.mqtt.homeassistant/src/main/resources/OH-INF/config/config.xml

```diff
@@ -4,14 +4,14 @@
 	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
 		https://openhab.org/schemas/config-description-1.0.0.xsd">
-	<config-description uri="binding:mqtt.homeassistant">
+	<config-description uri="binding:mqtt:homeassistant">
```

Comment:
Don't think this is expected, as it is the 'special' binding Id with a `.`

Reply:
Checked all other 93 files, all look good.

Reply:
@lsiepel I was not sure about that as well, but the other descriptions use the colon as well:
![image](https://github.com/user-attachments/assets/56968704-ed5f-4b17-84b4-b73667615436)

Reply:
@ccutrer wdyt?

Reply:
thanks for the feedback, I reverted this change